providers/Tidal/mod.ts

```diff
@@ -0,0 +1,330 @@
+import { availableRegions } from './regions.ts';
+import { type CacheEntry, MetadataProvider, type ProviderOptions, ReleaseLookup } from '@/providers/base.ts';
+import { DurationPrecision, FeatureQuality, FeatureQualityMap } from '@/providers/features.ts';
+import { parseHyphenatedDate, PartialDate } from '@/utils/date.ts';
+import { ResponseError } from '@/utils/errors.ts';
+import { encodeBase64 } from 'std/encoding/base64.ts';
+
+import type { Album, AlbumItem, ApiError, Image, Resource, Result, SimpleArtist, TokenResult } from './api_types.ts';
+import type {
+	ArtistCreditName,
+	Artwork,
+	EntityId,
+	HarmonyMedium,
+	HarmonyRelease,
+	HarmonyTrack,
+} from '@/harmonizer/types.ts';
+import { Label } from '../../harmonizer/types.ts';
+import { plural } from '../../utils/plural.ts';
+
+// See https://developer.tidal.com/reference/web-api
+
+const tidalClientId = Deno.env.get('HARMONY_TIDAL_CLIENT_ID') || '';
+const tidalClientSecret = Deno.env.get('HARMONY_TIDAL_CLIENT_SECRET') || '';
+
+export default class TidalProvider extends MetadataProvider {
+	constructor(options: ProviderOptions = {}) {
+		super({
+			rateLimitInterval: 1000,
+			concurrentRequests: 2,
+			...options,
+		});
+	}
+
+	readonly name = 'Tidal';
+
+	readonly supportedUrls = new URLPattern({
+		hostname: '{www.}?tidal.com',
+		pathname: String.raw`/browse/:type(album|artist)/:id(\d+)`,
+	});
+
+	readonly features: FeatureQualityMap = {
+		'cover size': 1280,
+		'duration precision': DurationPrecision.SECONDS,
+		'GTIN lookup': FeatureQuality.GOOD,
+		'MBID resolving': FeatureQuality.PRESENT,
+		'release label': FeatureQuality.BAD,
```

Comment:
I see. Yes, we should re-investigate once we find an example. Another possibility is that the API no longer returns this data and only the documentation  and schema still list the fields.

So another option would be to remove support for this field again. I just thought it might be easier to figure out if there are releases returning this data if we at least somewhat support it.